insights/specs/default.py

```diff
@@ -133,6 +133,7 @@ class DefaultSpecs(Specs):
     abrt_status_bare = simple_command("/usr/bin/abrt status --bare=True")
     alternatives_display_python = simple_command("/usr/sbin/alternatives --display python")
     amq_broker = glob_file("/var/opt/amq-broker/*/etc/broker.xml")
+    ansible_host = simple_file("ansible_host")
```

Comment:
Remove this spec and add it to `insights_archive.py` to support reading non-core-collection archives.